ccd-definition/SearchInputFields.json

```diff
@@ -5,5 +5,12 @@
     "CaseFieldID": "[CASE_REFERENCE]",
     "Label": "CCD Case Number",
     "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "legacyCaseReference",
+    "Label": "CaseMan reference number",
```

Comment:
Shouldn't it say `Caseman`? Also, did we confirm the specific label wording with business?

Reply:
labels are same as in case field, have shown to Darren, he is fine with this.